docs/example_workflows/workflow_with_cache.yml

```diff
@@ -0,0 +1,20 @@
+name: My workflow
+on:
+  push:
+  pull_request:
+jobs:
+  check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup Starknet Foundry
+        uses: foundry-rs/setup-snfoundry@v3
+
+      - name: Setup Scarb
+        uses: software-mansion/setup-scarb@v1
+        with:
+          scarb-lock: ./hello_starknet/Scarb.lock
```

Comment:
```suggestion
          scarb-lock: ./path/to/Scarb.lock
```

Reply:
It seems to me that example with dummy package is more practical. `hello_starknet` is the most basic package used in our docs so it's more practical imo.